charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -126,7 +126,7 @@ gloo:
       sslConfig:
         secretRef:
           # gloo.virtualservice.spec.sslConfig.secretRef.name -- Name of the secret containing the certificate information for this deployment.
-          name: gloo-public-tls
+          name: gloo-public-tlsC
```

Comment:
Why is the secret name changed?

Reply:
I changed it by accident and reverted it a couple minutes later